packages/clay-css/src/images/icons/angle-down-small.svg

```diff
@@ -4,6 +4,4 @@
 *
 * SPDX-License-Identifier: BSD-3-Clause
 -->
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
-	<path class="lexicon-icon-outline" d="M268,339.9l142.6-142.2c16.4-16.4-8.5-41.7-25.1-25.1L256,301.7L126.5,172.5c-16.8-16.7-41.6,8.7-25.1,25.1L244,339.9C251.2,347,261.4,346.4,268,339.9z"/>
-</svg>
\ No newline at end of file
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><path d="m8.672 10.56 4.05-4.11c.793-.844-.285-1.99-1.063-1.165L8 9.008 4.3 5.263c-.721-.768-1.781.415-1.056 1.186l4.06 4.12c.489.532.786.62 1.369-.01Z"/></svg>
```

Comment:
The border appears larger than it was.
<img width="630" alt="Screen Shot 2024-04-25 at 4 00 27 PM" src="https://github.com/liferay/clay/assets/13750819/70bae094-c448-4b8e-9f3c-51b8e45e6fe0">